packages/odyssey-icons/.svgo.yml

```diff
@@ -51,15 +51,16 @@ plugins:
   - removeEmptyContainers: true
   - mergePaths: true
   - removeUnusedNS: true
-  - sortAttrs: true
+  - sortAttrs: false
```

Comment:
@arnoldsandoval-okta FYI, we have to disable attr sorting if we want to use `addClassesToSVGElement`. The sorting plugin is bugged, so they don't play well together.

I figured the ease of class control trumps sorting (esp since they're all coming from the same tool).

Reply:
The [code for these plugins](https://github.com/svg/svgo/blob/master/plugins/sortAttrs.js) is [pretty minimal](https://github.com/svg/svgo/blob/master/plugins/addClassesToSVGElement.js). Could we look at contributing a fix upstream I like having sorted attrs :)